curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-roles-and-responsibilies/65d5d17a45be4e4d56be704a.md

```diff
@@ -11,7 +11,7 @@ dashedName: task-33
 
 The word `ever` is used in the present perfect tense to talk about experiences at any time up to now. It's often found in questions to ask if someone has done something at least once in their life.
 
-For example, `Have you ever visited Paris?` means `At any time in your life, did you visit Paris?`
+For example, `Have you ever visited Paris?` means at any time in your life, did you visit Paris?
```

Comment:
```suggestion
For example, `Have you ever visited Paris?` means "at any time in your life, did you visit Paris"?
```
Or something like that. It is correct to remove backticks, but the meaning needs to be grouped in some way